docs/README.md

```diff
@@ -16,7 +16,7 @@ Make a new "filename.md" within the category folder within `/docs/wiki/CATEGORY/
 ### Wiki style tips
 
 - Inline code highlighting (`$ echo 'this is inline'`) does not look the best in RTD, try to have as little inline syntax highlighting as possible.
-- **osqueryd**, **osqueryi** and other tool names should be in bold. Use `inline highligh` when a tool or script is mentioned for the first time.
+- **osqueryd**, **osqueryi** and other tool names should be in bold. Use `inline highlighτ` when a tool or script is mentioned for the first time.
```

Comment:
Not sure what kind of character set that is, but this is rendering as the greek Tao character to me. Mind changing this to be the en-us character set for `t`?
```
>>> 'highlighτ'
'highligh\xcf\x84'
```